tools/packaging/verify-deb-install.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+docker ps
+image="$(docker ps --filter ancestor=jrei/systemd-debian:12 --latest --format "{{.ID}}")"
+echo "Running on container: ${image}"
+
+dir="."
+if [ -n "${CI}" ]; then
+    dir="/drone/src"
+fi
+echo "Running on directory: ${dir}"
+
+cat <<EOF | docker exec --interactive "${image}" sh
+    # Install loki and check it's running
+    dpkg -i ${dir}/dist/loki_0.0.0~rc0_amd64.deb
```

Comment:
Shouldn't the binary have a version?

Reply:
oh dang, that'll fail on actual releases. Will fix